ap_src/templates/calendars/calendar.html

```diff
@@ -139,22 +139,22 @@ <h1 class="is-size-5 has-text-weight-bold" id="title-{{ item.team.name }}">{{ it
                             {% check_absences recurring_absence_dates|get_key:member.user.username year month_num day as check_recurring_absence %}
                             {% check_half_day half_days_data|get_key:member.user.username year month_num day as check_half_day %}
                             {% if check_recurring_absence %}
-                            <td style="background-color: #ffaa00;" class="CalendarCell" id="{{member.user.username}}/{{year}}-{{month_num}}-{{day}}/TRUE" data-absence-type="R"></td>
+                            <td data-editable="{{editable}}" style="background-color: #ffaa00;" class="CalendarCell" id="{{member.user.username}}/{{year}}-{{month_num}}-{{day}}/TRUE" data-absence-type="R"></td>
                             {% elif check_absence %}
-                            <td style="background-color: #ff0000;" class="CalendarCell" id="{{member.user.username}}/{{year}}-{{month_num}}-{{day}}/TRUE" data-absence-type="S"></td>
+                            <td data-editable="{{editable}}" style="background-color: #ff0000;" class="CalendarCell" id="{{member.user.username}}/{{year}}-{{month_num}}-{{day}}/TRUE" data-absence-type="S"></td>
                             {% elif check_half_day == "A" %}
-                            <td style="background: linear-gradient(to right, #ffffff 50%, #ff0000 50% );" class="CalendarCell" id="{{member.user.username}}/{{year}}-{{month_num}}-{{day}}/TRUE" data-absence-type="A"></td>
+                            <td data-editable="{{editable}}" style="background: linear-gradient(to right, #ffffff 50%, #ff0000 50% );" class="CalendarCell" id="{{member.user.username}}/{{year}}-{{month_num}}-{{day}}/TRUE" data-absence-type="A"></td>
                             {% elif check_half_day == "M" %}
-                            <td style="background: linear-gradient(to right, #ff0000 50%, #ffffff 50%);" class="CalendarCell" id="{{member.user.username}}/{{year}}-{{month_num}}-{{day}}/TRUE" data-absence-type="M"></td>
+                            <td data-editable="{{editable}}" style="background: linear-gradient(to right, #ff0000 50%, #ffffff 50%);" class="CalendarCell" id="{{member.user.username}}/{{year}}-{{month_num}}-{{day}}/TRUE" data-absence-type="M"></td>
                             {% else %}
                                 {% if day in bank_hol and bank_holidays.enabled %}
-                                    <td style = "background-color: {{ bank_holidays.colour }};" class="CalendarCell" id="{{member.user.username}}/{{year}}-{{month_num}}-{{day}}/FALSE"></td>
+                                    <td data-editable="{{editable}}"style = "background-color: {{ bank_holidays.colour }};" class="CalendarCell" id="{{member.user.username}}/{{year}}-{{month_num}}-{{day}}/FALSE"></td>
```

Comment:
1. Add a space before style
2. Remove the spaces around the equals style from `style = "background-color: {{ bank_holidays.colour }};"` so it should just be `style="background-color: {{ bank_holidays.colour }};"`